native/src/components/PageDetail.tsx

```diff
@@ -1,27 +1,55 @@
 import React, { ReactElement } from 'react'
 import styled from 'styled-components/native'
 
+import { InternalPathnameParser } from 'shared'
+
+import buildConfig from '../constants/buildConfig'
 import { contentDirection } from '../constants/contentDirection'
+import useNavigate from '../hooks/useNavigate'
 
-const Identifier = styled.Text`
-  font-family: ${props => props.theme.fonts.native.contentFontBold};
-  color: ${props => props.theme.colors.textColor};
-`
 const DetailContainer = styled.Text<{ language: string }>`
-  display: flex;
   flex-direction: ${props => contentDirection(props.language)};
   font-family: ${props => props.theme.fonts.native.contentFontRegular};
   color: ${props => props.theme.colors.textColor};
 `
 
+const Identifier = styled.Text`
+  font-family: ${props => props.theme.fonts.native.contentFontBold};
+  color: ${props => props.theme.colors.textColor};
+`
+
+const StyledButton = styled.Pressable`
+  /* Offset for bold sibling + underline */
+  margin-top: -2px;
+`
+
+const ButtonText = styled.Text`
+  color: ${props => props.theme.colors.linkColor};
+  text-decoration: underline;
+  text-decoration-color: ${props => props.theme.colors.linkColor};
+`
+
 type PageDetailProps = {
   identifier: string
   information: string
   language: string
+  locationPath?: string | null
```

Comment:
:wrench: Is there any reason this needs to be `location`-specific? Can't we just name it `path` or directly use `routeInformation`? If we choose to keep it location specific, the naming should be consistent and be `poiPath`.

Reply:
I just kept the name as we get it from the CMS but I can rename it :)